### DISCUSSION TITLE
a question about  json_parser

### DISCUSSION THREAD

**Discussion by loveyang2012** [vectordotdev/vector#17801] [Q&A]
I have gotten message like this:
{"log":"this is 456","tag":"demotag1","timestamp":"2023-06-29T22:08:58.758192934Z"}
{"log":"this is 123","tag":"demotag2","timestamp":"2023-06-29T22:08:58.580667480Z"}

when json_parser of transforms  is deprecated,how can I only get message like "this is 456" and  "this is 123",yes, only get json's value ,
and  I want sent "this is 456" to kafka1 ,"this is 123" to kafka2.

**Comment by dsmith3197** [comment]
Hi,

You can use the [remap transform](https://vector.dev/docs/reference/configuration/transforms/remap/) with the [`parse_json` function](https://vector.dev/docs/reference/vrl/functions/#parse_json) to transform your logs.

**Comment by dsmith3197** [comment]
Duplicate: https://github.com/vectordotdev/vector/discussions/17817
